docs/basics/markdown.md

```diff
@@ -26,6 +26,20 @@ span: 2
 
 Catalog provides many specimen types, from images, hints, code snippets to fully interactive HTML and React component specimens. For a full list of all specimens and their features, go to the [Specimen documentation](/specimens)
 
+## Links
+
+Use regular Markdown link to pages within your Catalog documentation.
+
+Each page section can be linked to directly by adding a hash to your link. Click on the link icon next to a heading to jump to that section.
+
+```code|lang-markdown
+- [A link to a page](/specimens)
+- [A link to a page section](/specimens#content)
+- [A link to section on the same page](#example-page)
```

Comment:
I'm actually pretty happy as it is now. I think it's the most intuitive way to write links to other pages (and sections) without having to update everything when you switch from browserHistory to hashHistory, or when you change the `basePath` (i.e. when you publish to a different subdirectories).

Of course it doesn't prevent link-rot, but I'm not sure that we can without using special syntax (which I'm averse to). If we want to add it later, nothing prevents us from doing it. And: if we would be consequent there, we'd have to check the page heading IDs too because these can change if someone changes their content. But we can not really do this without fundamentally changing how Catalog loads Markdown files (i.e. asynchronously) – which is something we could rethink but that's another story entirely.

Again, I'm all for making the whole editing experience more user-friendly and resilient to broken content but that should be part of a bigger coordinated effort.

What do you think?